src/ucp/wireup/wireup_ep.h

```diff
@@ -113,6 +113,12 @@ uct_ep_h ucp_wireup_ep_extract_next_ep(uct_ep_h uct_ep);
 
 uct_ep_h ucp_wireup_ep_extract_msg_ep(ucp_wireup_ep_t *wireup_ep);
 
+static inline int ucp_wireup_ep_is_next_ep_active(ucp_wireup_ep_t *wireup_ep)
+{
+    ucs_assert(wireup_ep->super.uct_ep != NULL);
+    return wireup_ep->aux_ep == NULL;
+}
```

Comment:
why converted it to inline?

Reply:
it is required in wireup.c:1447

Reply:
yes, but can we just define it as non-inline function?